object-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,19 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// Task 1:
+console.log("Task 1:");
+let GryffindorMembers = hogwarts.filter(({ house }) => house === "Gryffindor");
+GryffindorMembers.forEach(({ firstName, lastName }) => {
+  console.log('${firstName} ${lastName}');
+});
```

Comment:
Please, us back ticks (`) here like this:

console.log(\`${firstName} ${lastName}\`);